packages/core/src/execute.js

```diff
@@ -9,31 +9,27 @@ const createJSONtool = require('./tools/create-json-tool');
 const ensureArray = require('./tools/ensure-array');
 const injectInput = require('./http-middlewares/before/inject-input');
 const prepareRequest = require('./http-middlewares/before/prepare-request');
-const throwForStatus = require('./http-middlewares/after/throw-for-status');
 const ZapierPromise = require('./tools/promise');
 const { FORM_TYPE } = require('./tools/http');
 
 const constants = require('./constants');
 
 const executeHttpRequest = (input, options) => {
   options = _.extend({}, options, constants.REQUEST_OBJECT_SHORTHAND_OPTIONS);
-  return input.z
-    .request(options)
-    .then(throwForStatus)
```

Comment:
No more need to manually call `throwForStatus` after form-mode/short-hand requests.